server/aws-lsp-codewhisperer/src/language-server/netTransform/transformHandler.ts

```diff
@@ -338,7 +340,9 @@ export class TransformHandler {
                 getTransformAttempt = 0 // a successful polling will reset attempt
             } catch (e: any) {
                 const errorMessage = (e as Error).message ?? 'Error in GetTransformation API call'
-                this.logging.log('poll : error polling transformation job from the server: ' + errorMessage)
+                this.logging.log(
+                    'polling details : Error in polling transformation job from the server: ' + errorMessage
+                )
```

Comment:
We should be more explicit in our error messages and try to use as much standardized/common language as possible.

Example:
"getTransformation request failed: ${errorMessage}"

If this catch block spans more code than a single request, we should still update the language to keep it within the context of the business workflow rather than the logic itself, e.g. "An error occurred while waiting for transformation to complete: ..."